### DISCUSSION TITLE
在load数据时发现有很多线程在进行Knowhere_Build

### DISCUSSION THREAD

**Discussion by loomrain** [milvus-io/milvus#30681] [Q&A and General discussion]
请问这个是正常的情况吗？

**Comment by loomrain** [comment]
![image](https://github.com/milvus-io/milvus/assets/57178850/b30318be-9b15-4efd-bbde-6e522a6bfdb5)

**Comment by yhmo** [comment]
正常的。knowhere是milvus的索引和搜索引擎，它初始化的时候会构造两个线程池，一个用建索引任务，另一个是给搜索用的。线程的数量跟host的cpu核数相关(一般是等于cpu核数，使用DiskANN时是cpu核数的4倍）。
